### ISSUE TITLE
Opening the filter sidebar should focus the filter sidebar

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse-frontend#989]
## Description

Activating the filter sidebar should move the user's focus to the filter sidebar.

## Reproduction

1. Enable the filter sidebar
2. Observe that the page focus has _not_ shifted to the filter sidebar, and that one must tab through all search results to finally access the filters.

## Expectation

After pressing the filter button, the users focus should be moved to the filters sidebar. Likely the first element within the sidebar, or perhaps some kind of aria region, but this will require some research.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by sarayourfriend** [comment]
Useful resources:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/landmark_role

https://www.w3.org/TR/wai-aria-1.1/#landmark_roles

There are a lot of these we should be using for Openverse but the sidebar should probably use the [search landmark role](https://www.w3.org/TR/wai-aria-1.1/#search) in addition to the focus management described in this issue.
